src/module.c

```diff
@@ -3348,29 +3349,31 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  * * **cmdname**: The Redis command to call.
  * * **fmt**: A format specifier string for the command's arguments. Each
  *   of the arguments should be specified by a valid type specification:
- *   b    The argument is a buffer and is immediately followed by another
+ *
+ *   * b: The argument is a buffer and is immediately followed by another
```

Comment:
these don't look great in redis.io. maybe use similar formatting to what we have in RM_DictIteratorStartC ?

Reply:
You mean with backticks? Sure, I can do that.

Actually there are several small typos and small things that can be improved. I didn't want the diff to be too big, but I can sure do more if you want in a follow-up.

Reply:
I mean the  bbackttick, and more importantly the dashes.